catalog/plugin/catalog-plugin-metacard-validation/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -54,9 +54,22 @@
     </bean>
 
 
+    <!-- Post-Query Metacard Validity Filter Plugin -->
+    <bean id="post-query-plugin" class="ddf.catalog.metacard.validation.MetacardValidityFilterPlugin" >
+        <cm:managed-properties
+                persistent-id="ddf.catalog.metacard.validation.MetacardValidityFilterPlugin"
+                update-strategy="container-managed"/>
+        <property name="attributeMap">
+            <list />
+        </property>
+    </bean>
+
+
+
 
     <!-- Register in the OSGi Service Registry -->
     <service ref="pre-ingest-plugin" interface="ddf.catalog.plugin.PreIngestPlugin"/>
     <service ref="pre-query-plugin" interface="ddf.catalog.plugin.PreQueryPlugin"/>
+    <service ref="post-query-plugin" interface="ddf.catalog.plugin.PostQueryPlugin" ranking="100"/>
```

Comment:
ranking determines when a plugin is run in relation to others - for the ingest and pre-query we don't really care, but we need to be sure that the post-query plugin runs before the `FilterPlugin` to ensure proper Metacard security attribute